test/EvitaTokenTest.js

```diff
@@ -1,14 +1,26 @@
-const { expect } = require("chai");
-const { ethers } = require("hardhat");
-const hre = require("hardhat");
+const { expect } = require('chai');
+const { ethers, upgrades } = require('hardhat');
+const hre = require('hardhat');
+const IERC20Upgradeable = artifacts.require('IERC20Upgradeable');
 
-describe("EvitaToken", function () {
-  it("Should test something", async function () {
-    const EvitaToken = await hre.ethers.getContractFactory("EvitaToken");
-    const token = await EvitaToken.deploy(111);
+describe('EvitaToken', () => {
+  let token;
+  let owner;
 
-    await token.deployed();
+  beforeEach(async () => {
+    const EvitaToken = await ethers.getContractFactory('EvitaToken');
+    const evitaToken = await upgrades.deployProxy(EvitaToken, [
+      new IERC20Upgradeable('Evita', 'EVI'),
```

Comment:
Seems that error on this line:

```
Error: Provided address Evita is invalid, the capitalization checksum test failed, or it's an indirect IBAN address which can't be converted.
```

I have no any idea how to make it work

Reply:
`new IERC20Upgradeable('Evita', 'EVI')` returns object, not address
but we don't need it at all, pls see my comments